dashboard/src/components/ColoredCircle/ColoredCircle.tsx

```diff
@@ -17,7 +17,7 @@ const ColoredCircle = ({
     <div
       title={tooltipText}
       className={cn(
-        'inline-flex h-6 w-6 justify-center rounded-full px-1 text-black',
+        'inline-flex h-6 w-fit min-w-6 justify-center rounded-full px-1 text-black',
```

Comment:
why w0fit instead of w-6?

Reply:
we use the w-6 as a default, but there are some cases, mostly when the number is bigger than 999, that the w-6 is not larger enough to have the entire number inside it